drivers/sensor/adxl372/adxl372_trigger.c

```diff
@@ -0,0 +1,170 @@
+/*
+ * Copyright (c) 2018 Analog Devices Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <device.h>
+#include <gpio.h>
+#include <misc/util.h>
+#include <kernel.h>
+#include <sensor.h>
+#include "adxl372.h"
+
+static void adxl372_thread_cb(void *arg)
+{
+	struct device *dev = arg;
+	struct adxl372_data *drv_data = dev->driver_data;
+	const struct adxl372_dev_config *cfg = dev->config->config_info;
+	u8_t status1, status2;
+
+	/* Clear the status */
+	if (adxl372_get_status(dev, &status1, &status2, NULL) < 0) {
+		return;
+	}
+
+	if (drv_data->th_handler != NULL) {
+		/* In max peak mode we wait until we settle below the inactivity
+		 * threshold and then call the trigger handler.
+		 */
+		if (cfg->max_peak_detect_mode &&
+			ADXL372_STATUS_2_INACT(status2)) {
+			drv_data->th_handler(dev, &drv_data->th_trigger);
+		} else if (ADXL372_STATUS_2_INACT(status2) ||
+			ADXL372_STATUS_2_INACT(status2)) {
```

Comment:
Why are you OR-ing `ADXL372_STATUS_2_INACT(status2)` with itself?

Reply:
Good catch - This must be ADXL372_STATUS_2_ACTIVITY.